commandlineparser/include/Parser.h

```diff
@@ -0,0 +1,23 @@
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <sstream>
+#include <map>
+
+#include "Options.h"
+
+#ifndef PARSER_H
+#define PARSER_H
+
+namespace cmdParser
+{
+	class Parser 
+	{
+		std::map<std::string ,opts::Options*> command_list; 
```

Comment:
better to create std::map<std::string,std::shared_ptr<opts::Options>> command_list